### PR TITLE
Add Bootstrap and remove Carbon

### PR DESCRIPTION
**Pull request by derick-montague**

### CODE REVIEW

**Review by yoshiemuranaka — APPROVED** [review]

File: src/components/AppNavigation/AppNavigation.vue [hunk 1]
```diff
@@ -1,77 +1,31 @@
...
 
 <script>
-import Analytics16 from "@carbon/icons-vue/es/analytics/16";
-import DataCheck16 from "@carbon/icons-vue/es/data--check/16";
-import SettingsAdjust16 from "@carbon/icons-vue/es/settings--adjust/16";
-import Settings16 from "@carbon/icons-vue/es/settings/16";
-import Password16 from "@carbon/icons-vue/es/password/16";
+// import Analytics16 from "@carbon/icons-vue/es/analytics/16";
+// import DataCheck16 from "@carbon/icons-vue/es/data--check/16";
+// import SettingsAdjust16 from "@carbon/icons-vue/es/settings--adjust/16";
```
Comment (gtmills) [inline src/components/AppNavigation/AppNavigation.vue:17]: Why comment out? Why not just remove?

File: src/components/AppNavigation/AppNavigation.vue [hunk 2]
```diff
@@ -1,77 +1,31 @@
...
-    Analytics16,
-    DataCheck16,
-    SettingsAdjust16,
-    Settings16,
-    Password16
-  }
+  name: "AppNavigation"
+  // components: {
+  //   Analytics16,
+  //   DataCheck16,
```
Comment (gtmills) [inline src/components/AppNavigation/AppNavigation.vue:25]: Why comment out? Why not just remove?

File: package.json [hunk 3]
```diff
@@ -9,34 +9,33 @@
...
-    "@carbon/icons-vue": "^10.6.1",
-    "@carbon/themes": "^10.7.0",
-    "@carbon/vue": "^2.18.0",
-    "axios": "^0.19.0",
-    "carbon-components": "^10.7.4",
-    "core-js": "^3.3.2",
-    "vue": "^2.6.10",
-    "vue-router": "^3.1.3",
-    "vuex": "^3.0.1"
+    "@carbon/icons-vue": "10.6.1",
```
Comment (gtmills) [inline package.json:12]: Still using the carbon icons?

**Review by gtmills — APPROVED** [review]
Looks reasonable for a prototype

**Review by dixsie — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (derick-montague) [reply package.json:12]: Yes. Bootstrap uses 3rd party icons

File: src/components/AppNavigation/AppNavigation.vue (hunk 1 above)
Comment (derick-montague) [reply src/components/AppNavigation/AppNavigation.vue:17]: I will be pulling them back in for the navigation

File: src/components/AppNavigation/AppNavigation.vue (hunk 2 above)
Comment (derick-montague) [reply src/components/AppNavigation/AppNavigation.vue:25]: I will be pulling them back in for navigation.

**Comment by derick-montague** [comment]
Ingegrated into webui-vue in https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/28595/2

**Comment by derick-montague** [comment]
Ingegrated into webui-vue in https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/28595/2

**Pull request merged**